api/src/main/java/org/hawkular/apm/api/model/trace/Node.java

```diff
@@ -213,6 +218,20 @@ public Node setFaultDescription(String faultDescription) {
     }
 
     /**
+     * @return the properties
+     */
+    public Set<Property> getProperties() {
+        return properties;
```

Comment:
Do you intend to allow external objects to manipulate the contents of this set directly? If not, then it might be better to wrap the Set into a `Collections.unmodifiableSet()`